.github/workflows/snap-release.yaml

```diff
@@ -21,8 +21,8 @@ jobs:
           kata_url="https://github.com/kata-containers/kata-containers"
           latest_version=$(git ls-remote --tags ${kata_url}  | egrep -o "refs.*" | egrep -v "\-alpha|\-rc|{}" | egrep -o "[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+" | sort -V -r | head -1)
           current_version="$(echo ${GITHUB_REF} | cut -d/ -f3)"
-          # Check if the current tag is the latest tag
-          if echo -e "$latest_version\n$current_version" | sort -C -V; then
+          # Check if the current tag is the latest tag and it's not a alpha or rc release
+          if echo -e "$latest_version\n$current_version" | sort -C -V && ! echo "${current_version}" | egrep -q "\-alpha|\-rc"; then
```

Comment:
This seems reasonable, but I wonder if we should invert the logic so that we only build if the pattern does not match an `x.y.z` version for even more safety? Alternatively, since this is semver we could reject if the version string contains `-`.